python_transport/wirepas_gateway/dbus/sink_manager.py

```diff
@@ -259,6 +259,25 @@ def write_config(self, config):
 
         return res
 
+    @property
+    def cost(self):
+        try:
+            cost = self.proxy.SinkCost
+        except GLib.Error:
+            self.logger.error("Cannot get sink cost for sink {}".format(self.sink_id))
+            cost = 0
+        return cost
+
+    @cost.setter
+    def cost(self, new_cost):
+        if new_cost < 0 or new_cost > 254:
```

Comment:
Any chance it is new_cost is None? If so, the comparison will throw an exception